README.md

```diff
@@ -148,3 +148,7 @@ npm install
 ## Further help
 
 To get more help on the `angular-cli` use `ng --help` or go check out the [Angular-CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
+
```

Comment:
there is a lot not useful info or wrong info on susper in README, Can you remove all those in this pull.